rmfuse/fuse.py

```diff
@@ -389,6 +390,17 @@ def parse_args():
     parser.add_argument('--version', action='version', version=VERSION)
     return parser.parse_args()
 
+def isconnected(host='1.1.1.1', port=80, timeout=1):
+    # timeout is expressed in seconds
+    try:
+        s = socket.create_connection((host, port), timeout)
+        if s is not None:
+            s.close()
+        return True
+    except OSError:
+        pass
+    return False
```

Comment:
If the `return True` above doesn't move, this can be moved inside of the `except` block.